_posts/2019-06-20-why-we-are-doing-what-we-do.md

```diff
@@ -1,19 +1,19 @@
 ---
 layout: post
 current: post
-cover:  image_0.png
+cover: image_0.png
 title: Why we are doing what we do
+description: In our inaugural blog post, my aim is to introduce how we see the world and what we care about. Regarding what we do, I will simply say that we are an open source software company concerned with security for cyber-physical systems - i.e. protecting the ‘real world’ stuff that is being connected to the internet like cars, trains, factories, implanted medical devices and of course smart toasters.
 date: 2019-06-20 10:00:00
 category: news
 author: Al Esmail
-tags: [Open Source]
+tags: [engineering,wott,privatelink,heroky postgres,vpc]
```

Comment:
privatelink? heroky?

Reply:
This was necessary to verify the correctness of the output array tags.